ext/mysqli/tests/fetch/mysqli_fetch_all_data_types_variation.phpt

```diff
@@ -13,6 +13,12 @@ require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
 
 $link = default_mysqli_connect();
 
+try {
+    mysqli_query($link, "DROP TABLE test_mysqli_fetch_all_data_types_variation");
+} catch (mysqli_sql_exception $e) {
+    // harmless, to avoid table already existing if the test gets wedged and then done again with persistent DB instance
```

Comment:
You didn't put any code in the `catch` block. Either put some code here or remove it. To avoid the exception you can adjust the SQL to have `IF EXISTS`